src/garage/replay_buffer/her_replay_buffer.py

```diff
@@ -67,27 +76,41 @@ def _her_sample_transitions(episode_batch, sample_batch_size):
                                       *transitions[k].shape[1:])
             for k in transitions.keys()
         }
-        assert (transitions['action'].shape[0] == sample_batch_size)
+        assert transitions['action'].shape[0] == sample_batch_size
         return transitions
 
     return _her_sample_transitions
 
 
 class HerReplayBuffer(ReplayBuffer):
-    """
-    This class implements HerReplayBuffer.
+    """This class implements HerReplayBuffer.
 
     It constructs hindsight examples using future strategy.
+
+    Args:
+        replay_k (float): Ratio between HER replays and regular replays
+        reward_fun (callable): Function to re-compute the reward with
+            substituted goals
+        kwargs: Keyword arguments for base class
+
     """
 
     def __init__(self, replay_k, reward_fun, **kwargs):
         self._sample_transitions = make_her_sample(replay_k, reward_fun)
         super(HerReplayBuffer, self).__init__(**kwargs)
 
     def sample(self, batch_size):
-        """Sample a transition of batch_size."""
+        """Sample a transition of batch_size.
+
+        Args:
+            batch_size (int): Batch size to sample.
+
+        Return:
+            dict[numpy.ndarray]: See _her_sample_transitions above.
```

Comment:
users will not see documentation for private methods.

Reply:
Looks like pylint checks for private methods and complains. Should we disable that?